recipes-samples/images/lmp-gateway-image.bb

```diff
@@ -53,16 +47,4 @@ CORE_IMAGE_BASE_INSTALL += " \
     strace \
     tcpdump \
```

Comment:
Can you also create another for the debug/development tools such as strace/tcpdump/vim?